docs/book/src/reference/labels_and_annotations.md

```diff
@@ -1,62 +1,62 @@
+| cluster.x-k8s.io/skip-remediation                                | It is used to mark the machines that should not be considered for remediation by MachineHealthCheck reconciler.  | User | Machines |
+| clusterctl.cluster.x-k8s.io/block-move                           | BlockMoveAnnotation prevents the cluster move operation from starting if it is defined on at least one of the objects in scope. Provider controllers are expected to set the annotation on resources that cannot be instantaneously paused and remove the annotation when the resource has been actually paused.   | Cluster API | Objects involved in cluster move operation |
+| clusterctl.cluster.x-k8s.io/delete-for-move                      | DeleteForMoveAnnotation will be set to objects that are going to be deleted from the source cluster after being moved to the target cluster during the clusterctl move operation. It will help any validation webhook to take decision based on it.  | Cluster API | Objects involved in cluster move operation |
+| clusterctl.cluster.x-k8s.io/skip-crd-name-preflight-check        | Can be placed on provider CRDs, so that clusterctl doesn't emit an error if the CRD doesn't comply with Cluster APIs naming scheme. Only CRDs that are referenced by core Cluster API CRDs have to comply with the naming scheme.      | Cluster API | Provider CRDs |
+| controlplane.cluster.x-k8s.io/kubeadm-cluster-configuration      | It is a machine annotation that stores the json-marshalled string of KCP ClusterConfiguration. This annotation is used to detect any changes in ClusterConfiguration and trigger machine rollout in KCP.    | User | Machines |
+| controlplane.cluster.x-k8s.io/remediation-for                    | It is a machine annotation that links a new machine to the unhealthy machine it is replacing.      | User | Machines |
+| controlplane.cluster.x-k8s.io/remediation-in-progress            | It is a KCP annotation that tracks that the system is in between having deleted an unhealthy machine and recreating its replacement.     | User | Machines |
+| controlplane.cluster.x-k8s.io/skip-coredns                       | It explicitly skips reconciling CoreDNS if set.   | User | Control plane machines |
+| controlplane.cluster.x-k8s.io/skip-kube-proxy                    | It explicitly skips reconciling kube-proxy if set.  | User | Control plane machines |
+| machine.cluster.x-k8s.io/certificates-expiry                     | It captures the expiry date of the machine certificates in RFC3339 format. It is used to trigger rollout of control plane machines before certificates expire. It can be set on BootstrapConfig and Machine objects. The value set on Machine object takes precedence. The annotation is only used by control plane machines.      | User | BootstrapConfig and Machine objects |
```

Comment:
```suggestion
| machine.cluster.x-k8s.io/certificates-expiry                     | It captures the expiry date of the machine certificates in RFC3339 format. It is used to trigger rollout of control plane machines before certificates expire. It can be set on BootstrapConfig and Machine objects. The value set on Machine object takes precedence. The annotation is only used by control plane machines.      | Cluster API/User | BootstrapConfig/Machine |
```